src/SFML/Graphics/Font.cpp

```diff
@@ -143,6 +144,15 @@ bool Font::loadFromFile(const std::string& filename)
         return false;
     }
 
+    // Load the stroker that will be used to outline the font
+    FT_Stroker stroker;
+    if (FT_Stroker_New(static_cast<FT_Library>(m_library), &stroker) != 0)
```

Comment:
The stroker resource is not released if one of the subsequent allocations (e.g. charmap) fails.